po/CMakeLists.txt

```diff
@@ -13,9 +13,8 @@ set(DOMAIN ${PROJECT_NAME})
 set(POT_FILE ${DOMAIN}.pot)
 file(GLOB PO_FILES *.po)
 
-add_custom_target(${POT_FILE}
+add_custom_command(OUTPUT ${POT_FILE}
```

Comment:
It seems this change is not necessary, please revert this line, as with the change there is now no target to build the template. Instead, this leaves a command definition that is never used. Removing only the `BYPRODUCTS` below is sufficient to resolve the ninja confusion. I guess the real issue here is a bug in cmake with the generator?